packages/victory-core/src/victory-util/textsize.ts

```diff
@@ -243,6 +246,72 @@ const _approximateTextHeightInternal = (text: string | string[], style) => {
   }, 0);
 };
 
+const _measureWithDOM = (
+  text: string | string[],
+  style?: TextSizeStyleInterface,
+): { width: number; height: number } => {
+  const element = document.createElementNS("http://www.w3.org/2000/svg", "svg");
+  element.setAttribute("xlink", "http://www.w3.org/1999/xlink");
+
+  const containerElement = document.createElementNS(
+    "http://www.w3.org/2000/svg",
+    "text",
+  );
+  element.appendChild(containerElement);
+
+  element.style.position = "fixed";
+  element.style.top = "-9999px";
+  element.style.left = "-9999px";
+
+  document.body.appendChild(element);
+
+  const lines = _splitToLines(text);
+  for (const [i, line] of lines.entries()) {
+    const textElement = document.createElementNS(
+      "http://www.w3.org/2000/svg",
+      "tspan",
+    );
+    const params = _prepareParams(style, i);
+    textElement.style.fontFamily = "arial";
+    textElement.style.transform = `rotate(${params.angle})`;
+    textElement.style.fontSize = `${params.fontSize}px`;
+    textElement.style.lineHeight = params.lineHeight;
+    textElement.style.fontFamily = params.fontFamily;
+    textElement.style.letterSpacing = params.letterSpacing;
+    textElement.textContent = line;
+
+    containerElement.appendChild(textElement);
+  }
+
+  const { width, height } = containerElement.getBBox();
+  const result = { width, height };
+
+  element.remove();
```

Comment:
I assume we create a lot of text elements; could we reuse the `element` between calls?  (obviously we'd have to remove the children here)